app/src/androidTest/java/com/github/epfl/meili/home/FacebookAuthenticationServiceTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.github.epfl.meili.home
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import com.facebook.AccessTokenCreator
+import com.facebook.Profile
+import com.github.epfl.meili.models.User
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.FixMethodOrder
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+@RunWith(AndroidJUnit4::class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+class FacebookAuthenticationServiceTest {
+    private lateinit var fauth: FacebookAuthenticationService
+
+    @Before
+    fun before() {
+        fauth = FacebookAuthenticationService()
+    }
+
+    @Test
+    fun testA() {
+        assert(fauth.getCurrentUser() == null)
+    }
+
+    @Test
+    fun testB() {
```

Comment:
Nitpick: could you change the names of the tests to something more descriptive of what is being tested?